code/_onclick/hud/alert.dm

```diff
@@ -627,6 +627,25 @@ so as to remain in compliance with the most up-to-date laws."
 	overlays += I
 
 /atom/movable/screen/alert/ghost/Click()
+	if(!..())
+		return
+	var/mob/living/carbon/human/target = usr
+	target.ghost()
+
+/atom/movable/screen/alert/ghost_xeno
+	name = "Ghost"
+	desc = "Would you like to ghost? You will be notified when your body is removed from the nest."
+	icon_state = "template"
+	timeout = 5 MINUTES // longer than any infection should be
+
+/atom/movable/screen/alert/ghost_xeno/Initialize(mapload)
```

Comment:
Why not just make this a subtype of the standard ghost alert so you don't need to redefine this?

Reply:
Because they call the parent proc, which would be an issue (For the click() proc at least)

Reply:
Nevermind you make a fair point standby dropping commit